test/e2e/storage/csi_volumes.go

```diff
@@ -361,7 +361,17 @@ func getTestPod(t testsuites.StorageClassTest, claim *v1.PersistentVolumeClaim,
 
 func testCSIDriverUpdate(f *framework.Framework, driver drivers.TestDriver) {
 	// NOTE: this test assumes that all CSI drivers in csiTestDrivers deploy exactly one DaemonSet.
-	It("should work after update", func() {
+	var namespace string
+	testName := "should work after update"
+	if driver.GetDriverInfo().FeatureTag == drivers.GcePDExternalFeatureTag {
+		// Default namespace
+		namespace = metav1.NamespaceDefault
+		testName = testName + " [Disruptive]"
```

Comment:
(also in k/k CI, disruptive == serial)

Reply:
Yes we will definitely need to make it serial, I'm thinking that just adding the `[Disruptive]` tag is enough since, as you said, disruptive == serial